conf/ocsci/enable-multi-storagecluster.yaml

```diff
@@ -0,0 +1,12 @@
+# config file enable multi-storagecluster deployment which install internal and external cluster in the same cluster
+DEPLOYMENT:
+  multi_storagecluster: True
+
+ENV_DATA:
+  storage_cluster_name: 'ocs-storagecluster'
+  external_storage_cluster_name: 'ocs-external-storagecluster'
+  external_storage_cluster_namespace: 'openshift-storage-extended'
+
+REPORTING:
+  polarion:
+    deployment_id: 'OCS-5503'
```

Comment:
Maybe as this is not production config, we should delete this section from here.